clients/src/main/java/org/apache/kafka/common/Cluster.java

```diff
@@ -46,6 +46,8 @@
     private final Map<Integer, List<PartitionInfo>> partitionsByNode;
     private final Map<Integer, Node> nodesById;
     private final ClusterResource clusterResource;
+    private final Map<String, Uuid> topicIds;
+    private final Map<Uuid, String> topicNames;
```

Comment:
Do we have a use case that requires both maps to be stored in Cluster for fast lookups? Or would it be sufficient to store it one-way and look up the single map for the other way?

Reply:
Thank you for your suggestions, the `topicNames` is not used so I just removed it. Maybe we will add them if it's necessary in a future use case.